lib/model/store.dart

```diff
@@ -226,7 +228,10 @@ class PerAccountStore extends ChangeNotifier {
   }
 
   void handleEvent(Event event) {
-    if (event is HeartbeatEvent) {
+    if (event is RealmEmojiUpdateEvent) {
```

Comment:
If you put this anywhere other than the top of the list, then it won't have to touch any extra lines :-)

I'm happy to have the heartbeat at the top, as it's a boring no-op that exists for the sake of the transport protocol. This could go right after, before AlertWordsEvent, to match the order in events.dart.